CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 ### Changes
 
+- Adds `HorizontalRule` component
```

Comment:
You want this to be in a separate section, like 
```
### Adds 
- Adds `HorizontalRule` component

### Changes 
- Changes `Breadcrumbs` story from `.tsx` to `.mdx`

Reply:
Fixed.